action.yml

```diff
@@ -199,6 +199,73 @@ runs:
           cd /tmp/skaffold
           skaffold config set collect-metrics false
           skaffold deploy
+
+          if [ $? -eq 0 ]; then
+            skaffold render > rendered.yaml
+
+            container_images=""
+            if yq -e '.spec.template.spec.containers[]?.image' "rendered.yaml" > /dev/null 2>&1; then
```

Comment:
In addition to containers lets also capture `initContainers` which are used for running containers on startup or for side car containers. https://kubernetes.io/docs/concepts/workloads/pods/init-containers/